src/vmeTest/config.properties

```diff
@@ -2,5 +2,5 @@
 #DPIPORT = 3490
 DPISOCKETPATH = /tmp/vipo_socket
 VANTIQ_BASEURL = https://dev.vantiq.com
-VANTIQTOKEN = KtaUNyWsplL-a6Lujaghp7Ky7_v4JfI1Et7LQLunwqI=
-#LOG_LEVEL = DEBUG
+VANTIQTOKEN = <replace with Vantiq access token>
+# LOG_LEVEL = DEBUG
```

Comment:
Out of curiosity -- why the extra space

Reply:
inadvertent. i enabled then commented out again with an extra space. thx for looking over the changes.